.docker/mongo/databaseInit/initialSetup.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+echo 'Creating application user and db'
+
+mongo ${MONGO_INITDB_DATABASE} \
+        --host localhost \
+        --port 27017 \
+        -u ${MONGO_INITDB_ROOT_USERNAME} \
+        -p ${MONGO_INITDB_ROOT_PASSWORD} \
+        --authenticationDatabase admin \
+        --eval "db.createUser({user: '${MONGO_INITDB_ROOT_USERNAME}', pwd: '${MONGO_INITDB_ROOT_PASSWORD}', roles:[{role:'dbOwner', db: '${MONGO_INITDB_DATABASE}'}, {role:'dbOwner', db: 'TEST_DB'}]});"
```

Comment:
The hardcoded DB name here should be swapped out

Reply:
good catch!